packages/utils/lib/telemetry/metrics.ts

```diff
@@ -78,7 +78,10 @@ export enum Types {
     GET_RECORDS_COUNT = 'nango.server.getRecords.count',
     GET_RECORDS_SIZE_IN_BYTES = 'nango.server.getRecords.sizeInBytes',
 
-    CONNECTIONS_COUNT = 'nango.server.connections.count'
```

Comment:
doesn't have anything to do with server. it just happens to run  there (for now)

Reply:
I think we should avoid prefix with the services in general 👌🏻